automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/monitor/SubmissionDetailsPage.scala

```diff
@@ -60,6 +61,7 @@ class SubmissionDetailsPage(namespace: String, name: String, var submissionId: S
 
   def waitUntilSubmissionCompletes(): Unit = {
     while (!isSubmissionDone) {
+      Thread sleep 1000  // this is needed for tests to pass on alpha. if you want to change how this is done, make sure it passes on alpha env
```

Comment:
Do we know why this is only showing up as a problem on alpha? Is there a solution that doesn't involve `Thread sleep` that would work for alpha and also be more technically correct for all environments? If the timing doesn't work for one environment, we may see the same instability in other environments some day.

Reply:
I think that is an excellent question. but i have not looked into it. i'll run it by isaac to see if he knows why.